src/nodes.coffee

```diff
@@ -2509,7 +2509,8 @@ exports.Op = class Op extends Base
     return @compileUnary        o if @isUnary()
     return @compileChain        o if isChain
     switch @operator
-      when '?'  then @compileExistence o
+      when '?'  then @compileExistence o      # Check if not null and not undefined
+      when '??' then @compileExistence o, yes # Check only if not undefined
```

Comment:
I’ve refactored this to use the new only-not-undefined existence check if the right-hand value has an `isDefaultValue` property set. This meant adding a new property to `Value`, but no new operator. 60f046e313d0474feb5afa9b0453723c991eb793